change/@microsoft-teams-js-adac1a3d-5454-4c76-b1ef-d65e3f0c48c5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add showProfile API",
```

Comment:
Please use past tense for change file comments and backticks for APIs. (I have a PR out for updating the CONTRIBUTING file including this info in it. :)).

In this case, I think the preferred message would be "Added \`profile.showProfile\` API and unit tests"